README.md

```diff
@@ -6,7 +6,10 @@ The MoveIt Motion Planning Framework for **ROS 2**. For ROS 1, see [MoveIt 1](ht
 
 ## Continuous Integration Status
 
-[![Format](https://github.com/ros-planning/moveit2/actions/workflows/format.yml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2/actions/workflows/format.yml?branch=main) [![BuildAndTest](https://github.com/ros-planning/moveit2/actions/workflows/industrial_ci_action.yml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2/actions/workflows/industrial_ci_action.yml?branch=main) [![codecov](https://codecov.io/gh/ros-planning/moveit2/branch/main/graph/badge.svg?token=W7uHKcY0ly)](https://codecov.io/gh/ros-planning/moveit2)
+
+[![Formatting (pre-commit)](https://github.com/ros-planning/moveit2/actions/workflows/format.yaml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2/actions/workflows/format.yaml?query=branch%3Amain)
```

Comment:
I just randomly tried this link and got "No results matched your search". Somehow all workflows are listed twice on that page and when selecting the other one (the one not pointed to by default), the search "branch:main" shows all the results.  We could use "https://github.com/ros-planning/moveit2/actions?query=branch%3Amain+workflow%3AFormat" to fix this.

Reply:
The reason you get no results is that I renamed the file from `format.yml` to `format.yaml` in this PR and that isn't on main yet.

Reply:
:man_facepalming: